example/project-nextjs-tailwind/components/colors.js

```diff
@@ -0,0 +1,243 @@
+const Colors = () => {
+  return (
+    <div className="flex flex-col items-center w-3/4 pb-3 mb-5 border-b-2">
+      <p className="mb-3 h4 text-type-black-secondary">Colors</p>
+      <div className="flex flex-col items-center mb-3">
+        <p className="h5">Primary</p>
```

Comment:
Just for semantics, can you update the tags that use a heading css component to be the right html tag?

in this case: 
```suggestion
        <h5 className="h5">Primary</h5>
```